### ISSUE TITLE
Make hit counter for redirects better configurable

### ISSUE THREAD

**Issue by Sebobo** [neos/redirecthandler-ui#11]
I think it makes sense to have the hit counter on certain types of redirects.
Either based on the status code or actually configured for each redirect separately.

Having it active by default might be an issue for sites with lots of traffic.

**Comment by Benjamin-K** [comment]
This is not an issue of the UI itself and is already added in [`Neos.RedirectHandler`](https://github.com/neos/redirecthandler/blob/master/Configuration/Settings.yaml#L5):

```yaml
Neos:
  RedirectHandler:
    features:
      hitCounter: true
```

What should be added to the UI is the output of the hitcounter itself and the last hit date. This could also be added on a separate page.

**Comment by Sebobo** [comment]
Yes I know. We have and will have more issues than relate to all redirect related packages.
I'm trying to connect them as good as possible.

In the first step I plan to add the output with the last hit date maybe on hover or similar.
Then, if we want to have it configurable for each redirect (which people are interested in), we first need a change in Neos.RedirectHandler.DatabaseStorage and then here.

**Comment by paavo** [comment]
Having a hit counter sounds awesome to me 🤩 
Was there any progress in the last years?

**Comment by Sebobo** [comment]
no
